test/test_resign.py

```diff
@@ -7,79 +6,85 @@
 
 def test_resign(tmp_path):
     # copy to tmp path for safe ownership check
-    shutil.copyfile("good-bundle.raucb", tmp_path / "good-bundle.raucb")
+    in_bundle = tmp_path / "in.raucb"
+    shutil.copyfile("good-bundle.raucb", in_bundle)
 
+    out_bundle = tmp_path / "out.raucb"
     out, err, exitcode = run(
         "rauc"
         " --cert openssl-ca/dev/autobuilder-1.cert.pem"
         " --key openssl-ca/dev/private/autobuilder-1.pem"
         " --keyring openssl-ca/rel-ca.pem"
-        f" resign {tmp_path}/good-bundle.raucb {tmp_path}/out.raucb"
+        f" resign {in_bundle} {out_bundle}"
         " --signing-keyring openssl-ca/dev-only-ca.pem"
     )
-
     assert exitcode == 0
+    assert out_bundle.exists()
```

Comment:
A simple hint in the commit message that the refactoring is based on the fact that pytest's `tmp_path` fixture already is a `pathlib.Path` object would make it easier for others to follow.

Also slightly mixes variable declarations, internal path renames, and changes of `os.path.exists()` to `pathlib.Path. exists()` which isn't ideal but acceptable since it avoids touching some lines twice.
Some notes in the commit message might make it easier to get this, too.